src/db.js

```diff
@@ -31,4 +29,17 @@ getClient()
     console.error(err)
   })
 
-export { getClient, ObjectId }
+const DUMMY_TRIP_ID = '5bb8a294bc0c7b396d6b8abb'
+
+const getTrip = async (tripId = DUMMY_TRIP_ID) => {
+  let trips = await getClient().then(db =>
```

Comment:
Because you're using `async` you shouldn't use a `then()`! Instead form your code like this:
```javascript
const getTrip = async (tripId = DUMMY_TRIP_ID) => {
  const db = await getClient();
  const trips = await db.collection("trips").find({ _id: ObjectId(tripId) }, { limit: 1 }).toArray();
  // this is just nicer than trips[0]
  const [firstTrip] = trips;
  return firstTrip
}
```

Reply:
Also, why not using `findOne()`?